OneSignalSDK/onesignal/src/main/java/com/onesignal/OSNotificationRestoreWorkManager.java

```diff
@@ -61,7 +61,8 @@ public NotificationRestoreWorker(@NonNull Context context, @NonNull WorkerParame
         public Result doWork() {
             Context context = getApplicationContext();
 
-            OneSignal.initWithContext(context);
+            if (OneSignal.appContext == null)
```

Comment:
It seems that `Worker` would always have a `Context` so I won't expect a `null` check is need here.

Reply:
This is to avoid a new OneSignal.initWithContext(context); while the first initWithContext was called but not finished yet